test/e2e/cluster_upgrade_runtimesdk.go

```diff
@@ -148,6 +149,36 @@ func clusterUpgradeWithRuntimeSDKSpec(ctx context.Context, inputGetter func() cl
 		err = input.BootstrapClusterProxy.GetClient().Create(ctx, responses)
 		Expect(err).ToNot(HaveOccurred(), "Failed to create the responses configmap")
 
+		By("Wait for test Extension Deployment to be running")
```

Comment:
Can we move this part into a local util func at ~ the end of the file?

Just to keep the actual test case relatively readable

Reply:
Found a better variant which reuses another func providing the same :-)

Reply:
Yup looks good, let's see